Automattic/newspack-popups
# Insert prompts block count basis

### All Submissions:






### Changes proposed in this Pull Request:



Closes #623.

### How to test the changes in this Pull Request:

1. Load this PR.
2. Create a new inline prompt.
3. Set the trigger to `Blocks count` from the right sidebar, and set the number of blocks to be rendered before inserting the prompt.
4. Load a post and observe the prompt is inserted after the number of blocks set in 2.
5. Smock test some edge cases, like having the prompt after a heading (it shouldn't be inserted), the prompt as the first/last element on the content.

### Other information:
